app/views/hyrax/base/_work_title.erb

```diff
@@ -10,7 +10,9 @@
       <% end %>
     </div>
     <div class="col-sm-4 text-right">
-      <%= render "show_actions", presenter: presenter %>
+      <% if index == 0 %>
```

Comment:
Do we need the empty `<div>` for additional items, or should the conditional wrap lines 12 and 16 as well?

Reply:
I don't think we need the additional divs since the `col-sm-4` renders (in my test) the proper block on the page.

Reply:
deleted my previous comment.. I misread your question. I do think we should have the empty div so that they maintain that column/spacing and alignment on the page. If that doesn't really matter, I'm happy to adjust the statement and wrap the whole thing.. LMK @no-reply